Oliver-ke/Teamwork
# #28947558 Implement user edit article feature

#### What does this PR do?
- allows users on teamwork app edit their article post

#### Description of Task to be completed?
- add edit function to article controller
- wrote test for edit feature
- add edit endpoint

#### How should this be manually tested?
- clone this repository
- run `npm install` to install all dependencies
- start the server by running `npm run start:dev`
- open postman on  POST `localhost:5000/api/v1/articles` to create and article
- copy the article id
- open PATCH `localhost:5000/api/v1/articles/<articleId>` with update
- you should get a success response 

#### Any background context you want to provide?
- ensure you have postgreSQL installed and working
- ensure you have environments setup

#### What are the relevant project board stories?
[#28947558](https://github.com/Oliver-ke/Teamwork/projects/1#card-28947558)

#### Screenshots (if appropriate)
none

#### Questions:
